domains/newGame/NewGameScreen.jsx

```diff
@@ -42,11 +39,12 @@ const NewGameScreen = ({ route }) => {
         case "error":
             return (
                 <View>
-                    <TextInput>{{ error }}</TextInput>
+                    <Text>{error.message}</Text>
                 </View>
             );
         case "loaded":
-            return <Game playerName={playerName} game={game}></Game>;
+            navigation.navigate("Game", { playerName, gameId: game.id });
```

Comment:
always put side effect inside a useEffect
In this case you can create a Redirect component with the redirect useEffect.